### PR TITLE
Adopt existing cpu frequency implementation over abseil one

### PR DESCRIPTION
**Pull request by Sidong-Wei**
Solving issue https://github.com/tensorflow/tensorflow/issues/41033
Remove the abseil internal method of retrieving cpu frequency, and use a Tensorflow native one instead.

### CODE REVIEW

**Comment by Sidong-Wei** [comment]
Hi @gunan , would you mind take a look at this PR? Thanks.

**Review by gunan — DISMISSED** [review]

**Comment by Sidong-Wei** [comment]
Sanity check fails with the following suggestion:
```
FAIL: buildifier found errors and/or warnings in above BUILD files.
buildifier suggested the following changes:
tensorflow/core/platform/default/BUILD:
271d270
<         "//tensorflow/core/platform:snappy.h",
273a273
>         "//tensorflow/core/platform:snappy.h",
exit status 1
Please fix manually or run buildifier <file> to auto-fix.
```
However, I could not see the difference here (including the whitespaces); besides, as could be seen from the file changes, I did not edit this line in this PR. Do you have any idea what is causing the error @gunan ? Thanks.

File: tensorflow/core/platform/default/BUILD [hunk 1]
```diff
@@ -269,6 +269,8 @@ cc_library(
         "//tensorflow/core/platform:mem.h",
         "//tensorflow/core/platform:numa.h",
         "//tensorflow/core/platform:snappy.h",
```
Comment (gunan) [inline tensorflow/core/platform/default/BUILD:271]: Please sort the list alphabetically:
        "//tensorflow/core/platform:numa.h",
        "//tensorflow/core/platform:profile_utils/cpu_utils.h",
        "//tensorflow/core/platform:profile_utils/i_cpu_utils_helper.h",
        "//tensorflow/core/platform:snappy.h",

Comment (Sidong-Wei) [reply tensorflow/core/platform/default/BUILD:271]: I will do that, thanks!

**Review by gunan — APPROVED** [review]

**Comment by Sidong-Wei** [comment]
It seems that one check fails but I could not see the error message. It will be great if anyone could share the error message with me so that I could try to fix it, thanks.

**Comment by gunan** [comment]
I also cannot see the issue.
@gbaned could you help us import this PR?

**Pull request merged**